### ISSUE TITLE
On a Salesforce lightning and aura datatable, NVDA narrates the first cell value repetitively when traversing the row.

### ISSUE THREAD

**Issue by ifeanyiagwu** [nvaccess/nvda#15304]
### Steps to reproduce:
Navigate to https://developer.salesforce.com/docs/component-library/bundle/lightning-datatable/example
Enable NVDA on windows and ensure speech is heard
Tab into the datatable
Use right arrow key to traverse the row and notice the first cell value is announced

### Actual behavior:
When navigating the first row left to right via right arrow key, the NVDA speech history reads:

table
[http://www.salesforce.com Name](http://www.salesforce.com%20name/) (0) row 2 Website column 3
Name (0) row header Name (0) row 2 Label column 2
[http://www.salesforce.com Name](http://www.salesforce.com%20name/) (0) row 2 Website column 3
(497) 943-5109 Name (0) row 2 Phone column 4
$68.00 Name (0) row 2 Balance column 5
Aug 24, 2023 Name (0) row 2 CloseAt column 6

Notice how "Name (0)" is the first cell value read along with every cell

### Expected behavior:
The first cell value should not be announced when navigating a row

#### NVDA version:
NVDA Version : 2023.1 (2023.1.0.27913)

#### Windows version: 
OS : Windows 10
Version : 21H2 Build 19044.2728
Browser : Version 111.0.5563.65 (Official Build) (64-bit)

**Comment by Adriani90** [comment]
I cannot reproduce this with NVDA 2023 2 Beta 3 in Firefox 116.0.2, Chrome 116 or Edge 115.
Which browser are you using?
In my case NVDA does not read the name 0 cell repeatedly.

**Comment by seanbudd** [comment]
Welcome @ifeanyiagwu ,

We are unable to process this issue as it stands.

- Please fill out the issue template fully. You left out a lot of important information from the issue.
- This is very likely a problem with the implementation of the application/website, as a starting point please submit feedback to the authors to let them know about the problem.

Please update the issue filling out the issue template properly and let us know when it is ready. Alternatively, open a new issue..

Kind Regards,
NV Access Software Developers
